backend/environment-lib/src/main/java/org/mltooling/core/lab/LabAdminApiClient.java

```diff
@@ -45,7 +45,7 @@ public String getEndpointUrl() {
   }
 
   @Override
-  public StatusMessageFormat checkWorkspace(String workspaceId) {
+  public SingleValueFormat checkWorkspace(String workspaceId) {
     return executeRequest(
         Unirest.get(getEndpointUrl() + METHOD_CHECK_WORKSPACE)
             .getHttpRequest()
```

Comment:
In line 53 (below) the type needs to be changed from StatusMessageFormat to SingleValueFormat

Reply:
Solved in [6838a86](https://github.com/SAP/machine-learning-lab/pull/26/commits/6838a86687c94f497d3e7cfaeb7f654c71c44c61)